client/src/app/site/projector/components/projector-list/projector-list.component.html

```diff
@@ -19,7 +19,7 @@ <h2 translate>Projectors</h2>
         Reference projector for current list of speakers:
     </span>
     <mat-form-field>
-        <mat-select [disabled]="!!editId" [value]="projectors[0].reference_projector_id" (selectionChange)="onSelectReferenceProjector($event)">
+        <mat-select [disabled]="!!editId" [value]="projectors.length ? projectors[0].reference_projector_id : null" (selectionChange)="onSelectReferenceProjector($event)">
```

Comment:
@FinnStutzenstein This was one of the two travis testing fails that popped up. Please confirm that m change doesn't destroy anything

Reply:
Should be ok.